compiler-rt/test/hwasan/TestCases/stack-underflow.c

```diff
@@ -0,0 +1,25 @@
+// RUN: %clang_hwasan -g %s -o %t && not %run %t 2>&1 | FileCheck %s
+
+// Stack histories currently are not recorded on x86.
+// XFAIL: target=x86_64{{.*}}
+
+__attribute((noinline)) void buggy() {
+  char c[64];
+  char *volatile p = c;
+  p[-2] = 0;
+}
+
+int main() {
+  buggy();
+  // CHECK: WRITE of size 1 at
+  // CHECK: #0 {{.*}} in buggy{{.*}}stack-underflow.c:[[@LINE-6]]
+  // CHECK: Cause: stack tag-mismatch
```

Comment:
I am not sure about the output format. It has two `Cause:` clauses that are different to what this used to be. The second `Cause` is a more detailed description of the first, not an entirely different cause.

Reply:
The first one is true, the second one is "Potentially"